changelogs/fragments/2348-composer-no-interaction-option-discovery-to-avoid-hang.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - composer - add no-interaction option when discovering available options to avoid an issue where composer hangs (https://github.com/ansible-collections/community.general/pull/2348).
```

Comment:
```suggestion
  - composer - use ``no-interaction`` option when discovering available options to avoid an issue where composer hangs (https://github.com/ansible-collections/community.general/pull/2348).
```